README.md

```diff
@@ -59,9 +59,10 @@ Installation
         ]
     ```
 
-6.  Sync the database.
+6.  Migrate the database.
     ```sh
-        $ python manage.py syncdb
+        $ python manage.py migrate
+        # or python manage.py syncdb is using Django 1.6
```

Comment:
Instead of the Django 1.6 comment containing the syncdb command, we could separate the command out and specify more like:
For Django >= 1.7:
 `$ python manage.py migrate`
For Django 1.6:
 `$ python manage.py syncdb`